--- a/test/Makefile.in
+++ b/test/Makefile.in
@@ -1042,8 +1042,6 @@
 	litoct.awk \
 	litoct.in \
 	litoct.ok \
-	localenl.ok \
-	localenl.sh \
 	longsub.awk \
 	longsub.in \
 	longsub.ok \
@@ -1923,7 +1921,7 @@
 	zero2 zeroe0 zeroflag
 
 UNIX_TESTS = \
-	fflush getlnhd localenl pid pipeio1 pipeio2 poundbang rtlen rtlen01 \
+	fflush getlnhd pid pipeio1 pipeio2 poundbang rtlen rtlen01 \
 	space strftlng
 
 GAWK_EXT_TESTS = \
@@ -2044,7 +2042,7 @@
 NEED_CSV = csv1 csv2 csv3 csvodd
 
 # Lists of tests that run a shell script
-RUN_SHELL = exit fflush localenl modifiers next randtest rtlen rtlen01
+RUN_SHELL = exit fflush modifiers next randtest rtlen rtlen01
 
 # Lists of tests that need particular locales
 NEED_LOCALE_C = \
@@ -4704,10 +4702,6 @@
 	@-AWKPATH="$(srcdir)" $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
 	@-$(CMP) "$(srcdir)"/$@.ok _$@ && rm -f _$@
 
-localenl:
-	@echo $@
-	@-$(LOCALES) AWK="$(AWKPROG) $(GAWK_TEST_ARGS)" "$(srcdir)"/$@.sh  > _$@ 2>&1 || echo EXIT CODE: $$? >>_$@
-	@-$(CMP) "$(srcdir)"/$@.ok _$@ && rm -f _$@
 
 rtlen:
 	@echo $@
