--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -778,8 +778,6 @@
 	litoct.awk \
 	litoct.in \
 	litoct.ok \
-	localenl.ok \
-	localenl.sh \
 	longsub.awk \
 	longsub.in \
 	longsub.ok \
@@ -1659,7 +1657,7 @@
 	zero2 zeroe0 zeroflag
 
 UNIX_TESTS = \
-	fflush getlnhd localenl pid pipeio1 pipeio2 poundbang rtlen rtlen01 \
+	fflush getlnhd pid pipeio1 pipeio2 poundbang rtlen rtlen01 \
 	space strftlng
 
 GAWK_EXT_TESTS = \
@@ -1778,7 +1776,7 @@
 NEED_CSV = csv1 csv2 csv3 csvodd
 
 # Lists of tests that run a shell script
-RUN_SHELL = exit fflush localenl modifiers next randtest rtlen rtlen01
+RUN_SHELL = exit fflush modifiers next randtest rtlen rtlen01
 
 # Lists of tests that need particular locales
 NEED_LOCALE_C = \
