--- pc/Makefile.tst	2025-02-13 08:42:54.143477847 +0000
+++ "pc/Makefile copy.tst"	2025-02-13 14:19:02.130080151 +0000
@@ -460,18 +460,8 @@
 	$(MAKE) $(NEED_PMA) ; \
 	fi
 
-makepmafile: makepmafile.c
-	$(CC) $(srcdir)/makepmafile.c -o $@
-
-pma:
-	@echo $@
-	@GAWK_PERSIST_FILE=test.pma $(AWK) 'BEGIN { print ++i }' > _$@ 2>&1
-	@GAWK_PERSIST_FILE=test.pma $(AWK) 'BEGIN { print ++i }' >> _$@ 2>&1
-	@-$(CMP) "$(srcdir)"/$@.ok _$@ && rm -f _$@
-	@-$(RM) -f test.pma
-
 msg::
-	@echo ''
+	@echo 
 	@echo 'Any output from "$(CMP)" is bad news, although some differences'
 	@echo 'in floating point values are probably benign -- in particular,'
 	@echo 'some systems may omit a leading zero and the floating point'
